apps/api/src/routes/CandidateRoutes.ts

```diff
@@ -54,6 +56,13 @@ candidateRouter.post('/', async (req, res, next) => {
 
         const result = await candidateService.create(newCandidate);
 
+        const smsService = new SmsService(kafka);
+
+        smsService.send({
+            birthNumber: result.user.id,
+            message: 'Du har en melding om innsiktsarbeid på innbyggerpanelet.'
+        });
```

Comment:
Does this take into consideration that the CreateInsight component posts an array of candidates? Might have to implement some sort of bulk operation if not.